corehq/apps/export/templates/export/partials/delete_bulk_notice.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+<div data-bind="visible: bulkExportDownloadCount()">
+  <div style="float:right;" class="tooltip-bulkDelete" data-container=".tooltip-bulkDelete"
+       data-toggle="tooltip-bulkDelete" data-placement="top">
+    <input name="delete_list" type="hidden"/>
+    <p style="float:right">
+      <a class="btn btn-danger btn-primary"
+         data-toggle="modal"
+         data-bind="attr: {
+                      href: '#bulk-delete-export-modal'
+                    }">
+        <!-- ko if: isOData || isFeed -->
+          <!-- ko if: isMultiple -->
+            {% trans "Delete Selected Feeds" %}
+          <!-- /ko -->
+          <!-- ko if: !isMultiple() -->
+            {% trans "Delete Selected Feed" %}
+          <!-- /ko -->
+        <!-- /ko -->
+        <!-- ko if: !isOData && !isFeed -->
+          <!-- ko if: isMultiple -->
+            {% trans "Delete Selected Exports" %}
+          <!-- /ko -->
+          <!-- ko if: !isMultiple() -->
+            {% trans "Delete Selected Export" %}
+          <!-- /ko -->
+        <!-- /ko -->
+      </a>
+      {% include "export/dialogs/bulk_delete_custom_export_dialog.html" %}
+    </p>
+  </div>
+</div>
+
+<style>
+.tooltip-bulkDelete .tooltip .tooltip-inner {
```

Comment:
generally for CSS classes we avoid CamelCase, and use hyphens between words. so `tooltip-bulk-delete` would be a better practice. I'm curious though why you are choosing to use a custom tooltip here? It's best to avoid introducing custom one-off styles like this whenever possible (unless there is a really good reason) and better to stick to the stylesheet because it improves user familiarity with the UI and makes it easier to upgrade bootstrap versions. I don't remember custom tooltips being discussed in the spec, so I'm curious where this decision came from?

Reply:
> better to stick to the stylesheet because it improves user familiarity with the UI and makes it easier to upgrade bootstrap versions.
I agree! But this is the design that USH gave to Kirti as seen here: https://dimagi-dev.atlassian.net/browse/SS-176 (not specifically tooltips but popup text with color coordinated backgrounds)
I didn't think it was so uncommon but after looking at the stylesheet I think this might be the only instance where its being customized. Should I pass this suggestion to Kirti?

Reply:
Ah, ok...USH's designer is not familiar yet with the default styles in CommCare. You should always default to what is available in the current stylesheet unless a re-design of the whole site is happening. If a design comes back suggesting styles that are not already available in the stylesheet, you should always feel free to push back. Designs for the UIs are just suggestions and can be negotiated. ;-)

I think it's fine to use the normal bootstrap tooltips here. If they push back, I'll have a chat with them. We currently don't have a paradigm for styled tooltips and right before an upgrade to Bootstrap 5 is not really the best time to add new styles like this. :)

cc @orangejenny any idea why the USH designer is veering away from the standard UI elements we have available?

Reply:
the tooltips have been reverted back to their default styles with the exception of tooltip-bulk-export, which needed a slight margin or else it would've been awkwardly touching the page's edge.